test/bridge.test.ts

```diff
@@ -538,6 +556,28 @@ describe("Bridge", async function () {
     expect(BigNumber.from(l1InitialAUsdcBalance - 40n * USDC_UNIT)).to.be.lte(
       await aUsdc.balanceOf(l1user.address)
     );
+    //EXPECTED ENW BALANCES ON L2
```

Comment:
about format: we use lowercase in comments + there is a small typo

--> `// expected new balances on L2`